app.rb

```diff
@@ -0,0 +1,49 @@
+require "bundler/setup"
+require "pg"
+require "active_record"
+require "sinatra"
+require "sinatra/reloader"
+
+require_relative "db/connection"
+require_relative "models/apartment"
+require_relative "models/tenant"
+
+get '/' do
+  redirect to '/apartments'
+end
+
+get '/apartments' do
+  @apartments = Apartment.all
+  erb :"apartments/index"
+end
+
+get '/apartments/new' do
+  erb :"apartments/new"
+end
+
+get '/apartments/:id' do
+  @apartment = Apartment.find(params[:id])
+  erb :"apartments/show"
+end
+
+post '/apartments' do
+  @apartment = Apartment.create(params[:apartment])
+  redirect to ('/apartments/#{@apartment.id}')
+end
+
+get '/apartments/:id/edit' do
+  @apartment = Apartment.find(params[:id])
+  erb :"apartments/edit"
+end
+
+put '/apartments/:id' do
+  @apartments = Apartment.find(params[:id])
+  @apartment.update(params[:apartment])
+  redirect to ("/apartments#{@apartment.id}")
+end
+
+delete '/apartments/:id' do
+  @apartment = Apartment.find(params[:id])
+  @apartment.destroy
+  redirect to ('/apartments')
```

Comment:
apartment routes look good! but yea, as you mentioned, still need to add for tenants.